src/Nox.Ui.Blazor.Lib/Components/NoxTypes/EditStreetAddress.razor

```diff
@@ -0,0 +1,101 @@
+                      Clearable="true"
+                      Required="true"
+                      RequiredError="@ErrorRequiredMessage(TitleAddressLine1)"
+                      TextChanged="OnAddressLine1Changed"
+                      MaxLength="@MaxLengthAddressLine1"
+                      Value="CurrentAddressLine1" />
+    </MudItem>
+    <MudItem xs="6">
+        <MudTextField Label="@TitleAddressLine2"
+                      Variant="Variant.Outlined"
+                      Clearable="true"
+                      TextChanged="OnAddressLine2Changed"
+                      MaxLength="@MaxLengthAddressLine2"
+                      Value="CurrentAddressLine2" />
+    </MudItem>
+    <MudItem xs="6">
+        <MudTextField Label="@TitleRoute"
+                      Variant="Variant.Outlined"
+                      Clearable="true"
+                      TextChanged="OnRouteChanged"
+                      MaxLength="@MaxLengthRoute"
+                      Value="CurrentRoute" />
+    </MudItem>
+    <MudItem xs="6">
+        <MudTextField Label="@TitleLocality"
+                      Variant="Variant.Outlined"
+                      Clearable="true"
+                      TextChanged="OnLocalityChanged"
+                      MaxLength="@MaxLengthLocality"
+                      Value="CurrentLocality" />
+    </MudItem>
+    <MudItem xs="6">
+        <MudTextField Label="@TitleNeighborhood"
+                      Variant="Variant.Outlined"
+                      Clearable="true"
+                      TextChanged="OnNeighborhoodChanged"
+                      MaxLength="@MaxLengthNeighborhood"
+                      Value="CurrentNeighborhood" />
+    </MudItem>
+    <MudItem xs="6">
+        <MudTextField Label="@TitleAdministrativeArea1"
+                      Variant="Variant.Outlined"
+                      Clearable="true"
+                      TextChanged="OnAdministrativeArea1Changed"
+                      MaxLength="@MaxLengthAdministrativeArea1"
+                      Value="CurrentAdministrativeArea1" />
+    </MudItem>
+    <MudItem xs="6">
+        <MudTextField Label="@TitleAdministrativeArea2"
+                      Variant="Variant.Outlined"
+                      Clearable="true"
+                      TextChanged="OnAdministrativeArea2Changed"
+                      MaxLength="@MaxLengthAdministrativeArea2"
+                      Value="CurrentAdministrativeArea2" />
+    </MudItem>
+    <MudItem xs="6">
+        <MudTextField Label="@TitlePostalCode"
+                      Variant="Variant.Outlined"
+                      Clearable="true"
+                      Required="true"
+                      RequiredError="@ErrorRequiredMessage(TitlePostalCode)"
+                      TextChanged="OnPostalCodeChanged"
+                      MaxLength="@MaxLengthPostalCode"
+                      Value="CurrentPostalCode" />
+    </MudItem>
+
+    @if (CountrySelectionList != null && CountrySelectionList.Count > 0)
```

Comment:
Can we make CountrySelectionList non nullable list, empty by default, so we can just call CountrySelectionList .Any()